Library/Homebrew/cask/dsl.rb

```diff
@@ -316,9 +324,15 @@ def caveats(*strings, &block)
     end
 
     def discontinued?
+      # odeprecated "`discontinued?`", "`deprecated?` or `disabled?`"
```

Comment:
😍

Reply:
This is redundant since we already have the `odeprecated` for `caveats :discontinued`.

Reply:
Nevermind, `caveats :discontinued` is different from `@caveats?.discontinued?`.